docs/API/fliplet-datasources.md

```diff
@@ -173,6 +173,13 @@ connection.find({
   }
 });
 
+// Find a case insensitive match to the "name" column
```

Comment:
From this comment it isn't clear that this will also match values such as "Foo bob smith Bar". Can you amend the message as required?

Reply:
Given it's much more likely this will be used to match emails, I think it's best to use an email as example.

Reply:
Fair enough, will change to email example.